plugin.video.carnet-meduza/resources/lib/carnet-meduza.py

```diff
@@ -0,0 +1,270 @@
+# -*- coding: utf-8 -*-
+import os, sys
+import urllib, urlparse, json
+import xbmcgui, xbmcplugin, xbmcaddon
+import requests
+from register import device
```

Comment:
both of them are used. register/device is used on a first run to 'register' with the service and get api key. (mem)storage is used to store the api key as well to store pre-run check status (in memory, so it will run pre-run once until kodi is started again).